apps/etterlatte-beregning/src/main/kotlin/avkorting/Avkorting.kt

```diff
@@ -223,15 +241,15 @@ data class Avkorting(
         )
     }
 
-    fun beregnAvkortingRevurdering(
+    fun beregnAvkorting(
         virkningstidspunkt: YearMonth,
         beregning: Beregning?, // Kun null for forbehandling eteroppgjør
         sanksjoner: List<Sanksjon>,
     ): Avkorting {
         val virkningstidspunktAar = virkningstidspunkt.year
 
         val oppdaterteOppgjoer =
-            this.aarsoppgjoer.map { aarsoppgjoer ->
+            (this.aarsoppgjoer as List<AarsoppgjoerLoepende>).map { aarsoppgjoer ->
```

Comment:
```suggestion
            
        sjekk(this.aarsoppgjoer.none { it is Etteroppgjoer }) {
            "Kan ikke beregne avkorting hvis vi har etteroppgjør"
        } 
        val relevanteAarsoppgjoer = this.aarsoppgjoer.filterIsInstance<AarsoppgjoerLoepende>()
        val oppdaterteOppgjoer =
            relevanteAarsoppgjoer.map { aarsoppgjoer ->
```
eller noe i den dur? Vet ikke om feilmeldingen er den riktige for akkurat dette tilfellet

Reply:
Må bli en when is her